dev/code-generation/gen_ts_locales.py

```diff
@@ -35,6 +35,7 @@ class LocaleData:
     "es-ES": LocaleData(name="Español (Spanish)"),
     "fi-FI": LocaleData(name="Suomi (Finnish)"),
     "fr-FR": LocaleData(name="Français (French)"),
+    "fr-BE": LocaleData(name="Français (Belgique)"),
```

Comment:
The convention we're using here is "Localized Name (English Name)". Something like "Belgique Français (Belgian French"). I don't speak French so that translation is probably wrong, but you get the idea.

Reply:
I'm gonna go with "Belge (Belgian)" then :)